.github/workflows/ci.yml

```diff
@@ -42,7 +44,7 @@ jobs:
         with:
           target: ${{ matrix.target }}
           syspage: ${{ matrix.syspage }}
-          params: 'core fs test project image'
+          params: core fs test project image ${{ matrix.additional_params }}
```

Comment:
why did we remove quotation marks?

Reply:
There is the following error if I did it this way
![Screenshot from 2022-07-22 13-31-34](https://user-images.githubusercontent.com/77304431/180430820-f5829c22-32d8-48fc-a461-361e8dd85189.png)

Reply:
I've applied the previous approach and squashed the commits.

Reply:
Yeap but I think it may not work from ci-subcalls. If I understand how arguments are passed, we call the build script there as follows: `./phoenix-rtos-build/build.sh 'core fs test project image' 'test'`, and it will build only test in that case. I think we should just define default params in matrix and overwrite it in the `include` directive (as I remember that was the previous approach?)

Reply:
> Yeap but I think it may not work from ci-subcalls. If I understand how arguments are passed, we call the build script there as follows: `./phoenix-rtos-build/build.sh 'core fs test project image' 'ports'`, and it will build only test in that case. I think we should just define default params in matrix and overwrite it in the `include` directive (as I remember that was the previous approach?)

It works, I've re-triggered workflows here to be sure (https://github.com/phoenix-rtos/phoenix-rtos-tests/pull/118/checks). The previous attempt requires changing `required jobs` for all repos and if there were some changes in params we would have to update them (if I understand correctly)

Reply:
Yeah, it seems to be ok:
![Screenshot from 2022-07-22 14-17-56](https://user-images.githubusercontent.com/77304431/180437695-29504ccd-d959-4f76-9236-85ec3b1881ae.png)
![Screenshot from 2022-07-22 14-18-16](https://user-images.githubusercontent.com/77304431/180437711-b250aeff-5373-47e7-95cf-465de370951f.png)